django/db/models/query.py

```diff
@@ -423,13 +423,18 @@ def bulk_create(self, objs, batch_size=None):
         # this by using RETURNING clause for the insert query. We're punting
         # on these for now because they are relatively rare cases.
         assert batch_size is None or batch_size > 0
-        if self.model._meta.parents:
-            raise ValueError("Can't bulk create an inherited model")
+        # Checking all the parents share the same concrete model with the our
+        # model detects the inheritance pattern ConcreteGrandParent ->
+        # MultiTableParent -> ProxyChild. Simply checking self.model._meta.proxy
```

Comment:
There's an invalid case in the middle of `test_inheritance` and two at the end of `test_proxy_inheritance`. The `assertQuerySet` should come after both the valid and invalid cases (see a687b0b61ea5c3). I'd rather change the name of `test_inheritance` to `test_multi_table_inheritance`, i.e., the tests are organized by multi-table versus proxy rather than by valid or invalid. Thoughts?